src/app/globalsearch/globalsearch.component.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Search</title>
+</head>
+<body>
```

Comment:
you don't need to provide this, only html code that is relevant to the component itself. These tags are already provided here - https://github.com/FAANG/dcc-portal-frontend/blob/master/src/index.html

Reply:
We need to add the app-header component to this page to display FAANG toolbar